packages/widgetbook/README.md

```diff
@@ -16,8 +13,7 @@ A flutter package which helps developers cataloguing their widgets, testing them
 
 # See it in action!
 
-Check out the `Widgetbook` with the example app on our [github page](https://widgetbook.github.io). We also have introduction videos for [What is Widgetbook?](https://www.youtube.com/watch?v=vs3ocjMsl7s), [How to use Widgetbook?](https://www.youtube.com/watch?v=qcTZxJDLEAE) and [How to use Widgetbook generator?](https://www.youtube.com/watch?v=dh8hxgtbjtk).
-Furthermore, you can [check out the code of the app at github](https://github.com/widgetbook/widgetbook/tree/main/example). 
+Check out the `Widgetbook` with the example app on our [homepage](https://demo.widgetbook.io). Furthermore, you can [check out the code of the app at github](https://github.com/widgetbook/widgetbook/tree/main/example). 
 
```

Comment:
Check out how you can simplify your review processes with your designers, managers and clients with the example 'Widgetbook' hosted on 'Widgetbook Cloud' (link to demo). If you'd like to find our more about 'Widgetbook Cloud', please check out our website (link to website)